### PR TITLE
Ide 3717

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

File: maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenProjectBuilder.java [hunk 1]
```diff
@@ -257,6 +263,43 @@ else if (pluginName != null) {
...
+
+								configuration.addChild(url);
+
+								plugin.setConfiguration(configuration);
+
+								break;
+							}
+						}
+
+						mavenWriter.write(fileWriter, model);
```
Comment (gamerson) [inline maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenProjectBuilder.java]: Only write if you actually found the configuration to set.

Also what if there is already a configuration value?

File: maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenProjectBuilder.java [hunk 2]
```diff
@@ -257,6 +263,43 @@ else if (pluginName != null) {
 	}
 
 	public IStatus initBundle(IProject project, String bundleUrl, IProgressMonitor monitor) throws CoreException {
+		if (bundleUrl != null) {
```
Comment (gamerson) [inline maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenProjectBuilder.java]: We need to add unit tests for this.

**Comment by gamerson** [comment]
Closing see comments and please resend, thanks!

**Comment by jtydhr88** [comment]
@simonjhy @Camork see commits and please add unit tests and resend, thanks.

**Comment by Camork** [comment]
OK

**Comment by Camork** [comment]
Hi @gamerson ,i use the merge way to deal with url value.So no matter there have already a value presented or not, user choose to use a new url address so i merge it into a exited config. And if no present config ,it will create a new one.

**Pull request closed without merging**